### ISSUE TITLE
Lobby changes to assist tournament streamers

### ISSUE THREAD

**Issue by realitycheque** [mtgred/netrunner#5179]
It would be useful to have the status of the game in the list of active games. Even something like lobby/active/complete would  avoid picking finished tables when looking for something to watch on stream.

Current Score and/or time lapsed would also be good to see.

**Comment by NBKelly** [comment]
Just going to rephrase this as a checklist for later reference:
- [ ] Show status of active games (lobby/active/complete)
- [ ] Time Lapsed
- [ ] Current Score
